action-ssh-to-runner/action.yaml

```diff
@@ -84,6 +76,12 @@ runs:
         --------------------------------------------------
         EOF
         )
+
+        if [[ "${{ inputs.detached }}" == "true" ]]; then
+          echo "::notice title=SSH::Tmate started in detached mode. Connect using SSH: ${TMATE_SSH}"
```

Comment:
Could you add the following in the message printed in detached mode:
```
To connect you need:
- To be connected to the Scality VPN.
- The ssh key associated with your GitHub account.
```
I think it's important to print this info.

Reply:
added below